pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
```

Comment:
> https://registry.hub.docker.com/v1/repositories/openshift/origin/tags

All the tags seem to be there and less noisy. Each record only contains `name` and `layer` and would be fast to fetch.

> I am a bit worried that this will change/break.

Same here.